tests/mail_test.sh

```diff
@@ -140,6 +140,18 @@ function test_mail_parser()
   assert_equals_helper 'Invalid option passed' "$LINENO" "$ret" 22
 
   # valid options
+  parse_mail_options some -- extra -1 args HEAD^
+  expected='some extra -1 args HEAD^'
+  assert_equals_helper 'Set passthrough options' "$LINENO" "${options_values['PASSTHROUGH']}" "$expected"
+
+  parse_mail_options -1
```

Comment:
Test `-0`, `-10`, `-22`, `-100`?

Reply:
Ok. The current implementation only supports multiple digits after the `--`, but I will add more tests.

Reply:
Again, I think we should support -100 before `--` for usability sake.